zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -36,8 +36,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.gson.Gson;
-import com.google.gson.reflect.TypeToken;
+import java.util.*;
```

Comment:
Probably a nitpick but Zeppelin's [Java code conventions](https://github.com/apache/zeppelin/blob/master/CONTRIBUTING.md#code-convention) discourages [usage of wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).